core/state/statedb.go

```diff
@@ -474,6 +474,18 @@ func (s *StateDB) Suicide(addr common.Address) bool {
 	return true
 }
 
+func (s *StateDB) Selfdestruct6780(addr common.Address) bool {
```

Comment:
What is the meaning of the returnvalue here? I guess you  just mimicked the previous `Suicide`... but seems like it's very unused (and totally undocumented)

Reply:
Yeah, that appears to be the case.  neither `Suicide` nor `Selfdestruct6780` use the return value.